### PR TITLE
Support/1.x from master

### PR DESCRIPTION
**Pull request by goofballLogic**
I think I got everything but worth a check...

### CODE REVIEW

File: src/json-ld.net/Core/Context.cs [hunk 1]
```diff
@@ -14,15 +14,12 @@ namespace JsonLD.Core
...
-#if !PORTABLE && !IS_CORECLR
-        , ICloneable
-#endif
     {
         private JsonLdOptions options;
 
 		JObject termDefinitions;
 
-		public JObject inverse = null;
+		internal JObject inverse = null;
```
Comment (asbjornu) [inline src/json-ld.net/Core/Context.cs:22]: This is the sort of changes from #77 we don't want, no?

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: src/json-ld.net/Core/Context.cs (hunk 1 above)
Comment (goofballLogic) [reply src/json-ld.net/Core/Context.cs:22]: thanks for catching. I guess I screwed up the cherry-pick. Will fix this Monday.

Comment (goofballLogic) [reply src/json-ld.net/Core/Context.cs:22]: ....or tuesday....

**Review by asbjornu — APPROVED** [review]
LGTM! Would be great with a review from @sblom as well.

**Comment by goofballLogic** [comment]
@asbjornu shall we go ahead and merge this?

**Review by sblom — APPROVED** [review]
Looks right.

**Pull request merged**